config/locales/sv.yml

```diff
@@ -839,16 +837,19 @@ sv:
           h_branding_not_paid: "När detta är gjort behöver du göra färdigt din företagssida och betala din H-märkningsvagift. Mer information om hur du gör det kommer i ett mail efter att du betalt din medlemsavgift."
           thanks:  "Tack för att du vill vara medlem och bidrar till ett kvalitetssäkrat hundsverige."
 
-    user:
-      accepted:
-        subject: "Välkommen som medlem i Sveriges Hundföretagare"
+
+    member_mailer:
+      membership_granted:
+        subject: &subject_welcome "Välkommen som medlem i Sveriges Hundföretagare"
         greeting: *email_greeting
         message_text:
-          youre_active: "Du har nu aktiverats som medlem i systemet och kan logga in och hantera din profil och bygga upp din företagssida. När du är inloggad kan du redigera din företagssida här: "
+          welcome:  *subject_welcome
+          youre_active: "Ditt företag är nu H-märkt och du har aktiverats som medlem i systemet. Du kan logga in och hantera din profil och bygga upp din företagssida. När du är inloggad kan du redigera din företagssida här: "
```

Comment:
När du är inloggad kan du redigera din företagssida och betala din H-märkesavgift här: 

(or is there something else being invoked to tell them to pay? )

Reply:
At this point, they have already paid.  They get this email once they have satisfied _all of the requirements for membership_.  (So their application has been _approved_ and they have paid their membership fee.)

This email is separate from anything to do with their company.
(They have already received information about paying the company branding license fee (H-mäkt) [if needed] in an email they got after their application was approved.  See: 
https://github.com/AgileVentures/shf-project/wiki/Emails-automatically-sent-out